tests/e2e/visual/contacts/contact-user-management.wdio-spec.js

```diff
@@ -0,0 +1,150 @@
+const utils = require('@utils');
+const chtConfUtils = require('@utils/cht-conf');
+const path = require('path');
+const { resizeWindowForScreenshots, generateScreenshot, isMobile } = require('@utils/screenshots');
+
+describe('Contact and User Management', () => {
+  const updateContactSummarySettings = async () => {
+    await chtConfUtils.initializeConfigDir();
+    const contactSummaryFile = path.join(__dirname, 'config/contact-summary.templated.js');
+    const contactSummaryExtrasFile = path.join(__dirname, 'config/contact-summary-extras.js');
+    const { contactSummary } = await chtConfUtils.compileNoolsConfig({
+      contactSummary: contactSummaryFile,
+      contactSummaryExtras: contactSummaryExtrasFile
+    });
+    await utils.updateSettings(
+      { contact_summary: contactSummary},
+      { revert: true, ignoreReload: true, refresh: true, sync: true }
+    );
+    await commonPage.waitForPageLoaded();
+  };
+
+  const compileAndUploadForms = async () => {
+    await chtConfUtils.initializeConfigDir();
+    const formsPath = path.join(__dirname, 'forms');
+    await chtConfUtils.compileAndUploadAppForms(formsPath);
+  };
+
+  const docs = dataFactory.createHierarchy({
+    name: 'Janet Mwangi',
+    user: true,
+    nbrClinics: 10,
+    nbrPersons: 4,
+    useRealNames: true,
+  });
+
+  before(async () => {
+    await resizeWindowForScreenshots();
+    await utils.saveDocs([...docs.places, ...docs.clinics, ...docs.persons, ...docs.reports]);
+    await utils.createUsers([docs.user]);
+    await utils.addTranslations('en', {'contact.last.visit.unknown': 'Last Visited'});
+    await loginPage.login(docs.user);
+    await updateContactSummarySettings();
+  });
+
+  after(async () => {
+    await utils.deleteUsers([docs.user]);
+    await utils.revertDb([/^form:/], true);
+    await utils.revertSettings();
+  });
+
+  afterEach(async () => {
+    await commonPage.goToBase();
+  });
+
+  describe('Contact and user overview', () => {
+    it('should show contacts list, search, profiles (person and family), '+
+      'contact summary', async function() {
+      await commonPage.goToPeople();
+      expect(await commonPage.isPeopleListPresent()).to.be.true;
+      await generateScreenshot('people', 'list');
+      await searchPage.performSearch('Beatrice');
+      await generateScreenshot('people', 'search');
+      await searchPage.clearSearch();
+      await contactPage.selectLHSRowByText('Beatrice Bass Family');
+      await searchPage.clearSearch();
+      await generateScreenshot('people', 'profile-family');
+      await commonPage.goToBase();
+      await commonPage.goToPeople();
+      await contactPage.selectLHSRowByText('Beatrice Bass');
+      await generateScreenshot('people', 'profile-person');
+    });
+
+    it('should show condition cards', async function() {
+      if (!await isMobile()){
+        this.skip();
+      }
+      await commonPage.goToPeople();
```

Comment:
I tried this approach first, but for some reason, the variable was set to true when it was mobile, but the tests with the `(isMobile ? it : exit)` condition were still skipped.

Reply:
It seems it only works when the condition is defined under the `describe` (no in before or beforeEach) so it can determine the cases to include. 

it's fine leaving it as it is